torch/onnx/_internal/exporter.py

```diff
@@ -75,14 +82,179 @@ class ONNXFakeContext:
     """List of paths of files that contain the model `state_dict`"""
 
 
+class OnnxRegistry:
+    """Registry for ONNX functions.
+
+    The registry maintains a mapping from qualified names to symbolic functions under a
+    fixed opset version. It supports registering custom onnx-script functions and for
+    dispatcher to dispatch calls to the appropriate function.
+
+    Attributes:
+        _registry: The registry maps OpNameto a list of SymbolicFunctions. It is important
+            not to directly modify this variable. Instead, access to it should be done through
+            the public methods: register_custom_op, get_functions, and is_registered_op.
+
```

Comment:
```suggestion
    """Registry for ONNX operators.

    The registry maintains a mapping from PyTorch ATen operator qualified names to ONNX functions under a
    system-defined opset version. It supports registering custom operators.
```
Tried to NIT the docstring to hide implementation detail that the registry user shouldn't worry about.

Removed private stuff shouldn't be publicly documented. You can add a comment with this info somewhere, like where `self._registry` is initialized